google-pub/sample.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "Google MQTT Publish",
+    "category": "Cloud",
+    "tag": "cloud",
+    "date": "2018-2-21",
+    "description": "An application based on Google* Cloud IoT Core for publishing a single message. See the README.md file in the GitHub repo for this project before continuing.",
+    "platform": {
+	"libs":[
+		"Jannson",
+		"libjwt",
+		"OpenSSL",
+		"paho.mqtt.c",
+	],
+	"os": ["ubuntu-desktop-xenial-64bit"]
+    },
+    "service_provider": "Google Cloud IoT Core",
+    "service_provider_uri": "https://cloud.google.com/iot-core/m",
+    "service_provider_description": "A fully managed service to easily and securely connect, manage, and ingest data from globally dispersed devices",
+	"additional_fields":[
+		{
+		 "id":"deviceId",
+		 "name":"Device id",
+		 "description":"Registered device id"
+		},
+		{
+		 "id":"region",
+		 "name":"Region",
+		 "description":"The server region"
+		},
+		{
+		 "id":"registryId ",
+		 "name":"Registry Id",
+		 "description":"The registry id"
+		},
+		{
+		 "id":"projectId",
+		 "name":"Project ID",
+		 "description":"The project id"
+		}
+	]
```

Comment:
Two more parameters need to be added:
id:ecpath
name: Private EC Key File Path
description: Full Path to Private EC Key File, e.g. /etc/google-cloud-keys/ec_private.pem

id:rootpath
name: roots.pem file path
description: Full Path to roots.pem File, e.g. /etc/google-cloud-keys/roots.pem

Reply:
Done